env/includes/3dsmax/shot.yml

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2017 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+#
+
+# this configuration defines the behavior of the 3dsMax Integration when it
+# is running in its shot level configuration
+
+includes:
+- ../common/frameworks.yml
+- ../common/engines.yml
+- ../common/apps.yml
+- ../common/settings/tk-multi-publish2.yml
+
+3dsmax.shot:
+  apps:
+    tk-multi-about: '@common.apps.tk-multi-about'
+
+    tk-multi-pythonconsole: '@common.apps.tk-multi-pythonconsole'
+
+    tk-multi-setframerange: '@common.apps.tk-multi-setframerange'
```

Comment:
for each engine, shot and project are identical apart from the frame range app. Could we use includes similar to the common ones, to get rid of some of the duplication? E.g. something like `@houdini.apps.tk-multi-loader2`, and then that would be defined in a `houdini/apps.yml`? What do you think? That would make it consistent with the `common` structure.